src/AppInstallerCLICore/Argument.cpp

```diff
@@ -8,96 +8,100 @@
 
 namespace AppInstaller::CLI
 {
-    using namespace AppInstaller::CLI::Execution;
-    using namespace Settings;
+	using namespace AppInstaller::CLI::Execution;

```

Comment:
Your editor has changed spaces to tabs.  We use 4 spaces rather than a tab.  This is the case throughout your change (although auto-format hit this file really hard).

Reply:
I'm going to leave this comment here, but for the other items, as I check them off I'm not going to add additional comments. I'm not sure your practice but I usually address each comment with an acknowledgement of the comment and the change. but since there's > 10 items I'm going to forego them and leave this one comment separate the src/AppInstallerCLICore/Workflows/GetRequestedUrlFromManifest.cpp comment of the " Get URL's from Manifest" comment as it may not be clear what I've done to resolve it.  (and the fact that at this time, I haven't tested the changes 🔍 ) :)